src/pages/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import Home from './pageComponents/Home'
+
+export default () => {
+   return(
+    // <Layout>
+      <Home />
+    // </Layout>
+  )
+}
```

Comment:
Can you configure your text editor to leave trailing newlines at the end of the files? GitHub here is showing that you don't have a final newline with that red icon there.

There are some tools that will not function properly without the newline character at the end, so it's better to always leave it in. See https://github.com/airbnb/javascript#whitespace--newline-at-end. We can enforce this by either adopting the airbnb lint ruleset (which we do in the askdarcel-web app) or at least enabling the `eol-last` ESLint rule.

Reply:
Thanks. I added config to adopt the airbnb style